src/components/operations/operation-details/ko/runtime/graphql-console.ts

```diff
@@ -201,20 +194,20 @@ export class GraphqlConsole {
             // Go through every node in a new generated parsed graphQL, associate the node with the created tree from init and toggle checkmark.
             GraphQL.visit(ast, {
                 enter(node) {
-                    if (node.kind === "Field" || node.kind === "Argument" || node.kind === "ObjectField") {
+                    if (node.kind === "Field" || node.kind === "Argument" || node.kind === "ObjectField" || node.kind === "InlineFragment") {
```

Comment:
Javier, could you please move those to constants. There is a chance that they already defined by the library, if not, let's just add it.

Reply:
I mean all of those: `InlineFragment`, `Argument`, `Field`, `StringValue`, `Variable`, etc,

Reply:
Sure thing!